Projects/Authentication/Sources/AuthenticationStore.swift

```diff
@@ -0,0 +1,215 @@
+import Apollo
+import Flow
+import Foundation
+import Presentation
+import UIKit
+import hCore
+import hCoreUI
+import hGraphQL
+
+struct OTPState: StateProtocol {
+    var isLoading = false
+    var isResending = false
+    var id: String? = nil
+    var code: String = ""
+    var codeErrorMessage: String? = nil
+    var emailErrorMessage: String? = nil
```

Comment:
What do you think about adding a default `StateProtocol` property called `var errors: (Error) -> Void` or something of that nature? Then when we fetch anything we can `.onError` and return a default `setError` ?

Reply:
Ah wait I forgot properties in `State` have to be codable, equitable -> maybe var errors: [Error]